tex/LaTeXML/Portfile

```diff
@@ -4,11 +4,11 @@ PortSystem          1.0
 PortGroup           texlive 1.0
 
 name                LaTeXML
-version             0.8.6
-revision            2
-checksums           rmd160  3f71105b7da576ab73e714430626888080716602 \
-                    sha256  9529c651b67f5e8ddef1fd1852f974e756a17b711c46d4118f0677ad0e6e9bb1 \
-                    size    13776498
+version             0.8.7
+revision            1
```

Comment:
revision field should be reset to 0 when updating the version

Reply:
Of course; Sorry about that. Fixed now.